components/tyk-pump/templates/deployment-pmp.yaml

```diff
@@ -36,11 +36,9 @@ spec:
         image: "{{ .Values.pump.image.repository }}:{{ .Values.pump.image.tag }}"
         imagePullPolicy: {{ .Values.pump.image.pullPolicy }}
         workingDir: "/opt/tyk-pump"
-        {{- if has "prometheus" .Values.pump.backend }}
         ports:
-          - name: prom-metrics
-            containerPort: {{ .Values.pump.prometheusPump.port }}
-        {{ end }}
+          - name: tyk-pump-port
+            containerPort: {{ .Values.pump.containerPort }}
```

Comment:
should we add an if statement around this? if the service is enabled?

Reply:
@zalbiraw we'll create another ticket for the development. let's merge this first. thanks again for your support. i am closing this conversation to merge the PR. I'll link Jira ticket to this conversation.